src/variables.scss

```diff
@@ -60,11 +60,11 @@ $warning: #ffb105;
 $warning-dark: #e5a000;
 $warning-darker: #936620;
 
-$success-lighter: #fff4bf;
-$success-light: #ffe065;
-$success: #ffb105;
-$success-dark: #e5a000;
-$success-darker: #936620;
+$success-lighter: #dcfcdc;
+$success-light: #70e17b;
+$success: #00a91c;
+$success-dark: #33803f;
+$success-darker: #3c643b;
```

Comment:
In Figma these were showing the correct colors but the hex code labels were wrong. They've since been updated [here](https://www.figma.com/file/oGKbyCnCRRdNzLYbiags93/LDAF-Component-Library-USWDS-3.0.2?type=design&node-id=2305-12131&t=CRvVxEVyuShcrMuq-4).